src/main/java/gift/service/GiftService.java

```diff
@@ -0,0 +1,42 @@
+package gift.service;
+
+
+import gift.exception.DataNotFoundException;
+import gift.model.Gift;
+import gift.model.GiftDao;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class GiftService {
+
+    private GiftDao giftDao;
```

Comment:
final 키워드를 통해 불변으로 선언하면 더욱 좋을 것 같습니다ㅎㅎ
```suggestion
    private final GiftDao giftDao;
```

Reply:
final 키워드 추가하였습니다!